integration_test/take_survey_widget_test.dart

```diff
@@ -0,0 +1,138 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:get/get.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:survey/main.dart';
+import 'package:survey/managers/user_manager.dart';
+import 'package:survey/pages/ui/multiple_choice_rating.dart';
+import 'package:survey/pages/ui/nps_rating_bar.dart';
+import 'package:survey/pages/ui/slide_rating_bar.dart';
+import 'package:survey/pages/ui/smiley_rating_bar.dart';
+import 'package:survey/pages/ui/text_area_rating.dart';
+import 'package:survey/pages/ui/text_field_rating.dart';
+import 'package:survey/use_cases/create_response_use_case.dart';
+import 'package:survey/use_cases/get_login_state_use_case.dart';
+import 'package:survey/use_cases/get_survey_detail_use_case.dart';
+import 'package:survey/use_cases/get_surveys_use_case.dart';
+
+import 'fake/fake_create_survey_response_use_case.dart';
+import 'fake/fake_get_login_state.dart';
+import 'fake/fake_get_survey_detail_use_case.dart';
+import 'fake/fake_survey_use_case.dart';
+import 'fake/fake_user_manager.dart';
+
+main() {
+  Get.testMode = true;
+
+  group('Validate take survey flow', () {
+    final binding = IntegrationTestWidgetsFlutterBinding.ensureInitialized()
+        as IntegrationTestWidgetsFlutterBinding;
+
+    binding.framePolicy = LiveTestWidgetsFlutterBindingFramePolicy.fullyLive;
```

Comment:
does `framePolicy` make the test smoother? I feel like we are running a real app 🤪

Reply:
I think so 😂